src/utils/retry-error.rb

```diff
@@ -25,65 +24,44 @@
 #  to the completed folder, if fails it is not moved.
 
 # Make Sure that the ENV variables are set
-if (!ENV.include? 'HYDRA_PROJECT_NAME') then
+if !ENV.include? 'HYDRA_PROJECT_NAME'
   abort "Missing ENV variable, 'HYDRA_PROJECT_NAME'"
 end
 
 # We are expecting to run a single file again by hand.
 # Expecting this check to make sure that there is an argument given
-if ARGV.empty?
-  puts "Error: Expected filename"
-  abort "Usage: ruby ./retry-error.rb {filename.yaml}"
-end
-
-# Don't let them add more arguments than needed
-if ARGV.length >= 2
-  puts "Error: Too many arguments supplied"
-  abort "Usage: ruby ./retry-error.rb {filename.yaml}"
-end
+usage = 'Usage: ruby ./retry-error.rb {filename}'
+abort "Error: Expected filename.\n #{usage}" if ARGV.empty?
+abort "Error: Too many arguments.\n #{usage}" if ARGV.length > 1
 
 # define the filename from the arguments
 filename = ARGV
 
-# Mikes send notifications code
-def send_notifications(emailAddr, msgstr)
-  msgstr = "From: libsys@mail.wvu.edu\nSubject: Importing Update\n #{msgstr}"
-  Net::SMTP.start('smtp.wvu.edu', 25) do |smtp|
-    smtp.send_message msgstr, 'libsys@mail.wvu.edu', emailAddr
-  end
-end
-
-# Make sure directories are present and in vars for use later
-in_process_dir = "/mnt/nfs-exports/mfcs-exports/#{ENV['HYDRA_PROJECT_NAME']}/control/hydra/in-progress"
+# define directories
 error_dir = "/mnt/nfs-exports/mfcs-exports/#{ENV['HYDRA_PROJECT_NAME']}/control/hydra/error"
 success_dir = "/mnt/nfs-exports/mfcs-exports/#{ENV['HYDRA_PROJECT_NAME']}/control/hydra/finished"
 
 # if there is no file, abort the program
-if !File.exists?("#{error_dir}/#{filename}")
-  abort "File is missing make sure that it is correct and present in the error folder"
+if !File.exist?("#{error_dir}/#{filename}.yaml")
```

Comment:
since you appear to be adopting rubo cop this should be an unless instead of a negative if. also, one line.

`abort '#{filename} is missing make sure that it is correct and present in the error folder' unless File.exist?("#{error_dir}/#{filename}.yaml")`

This is another ruby style guide i don't agree with. I prefer the negative if ... but, one lining it with an if is good :)